File: web/src/app/models/employee.ts
export class Employee{
    
    private _Id : number;
    public get Id() : number {
        return this._Id;
    }
    public set Id(v : number) {
        this._Id = v;
    }
    
    private _Name : string;
    public get Name() : string {
        return this._Name;
    }
    public set Name(v : string) {
        this._Name = v;
    }
    
    private _Age : number;
    public get Age() : number {
        return this._Age;
    }
    public set Age(v : number) {
        this._Age = v;
    }
    
    private _ProjectId : number;
    public get ProjectId() : number {
        return this._ProjectId;
    }
    public set ProjectId(v : number) {
        this._ProjectId = v;
    }
    
    
    private _Salary : number;
    public get Salary() : number {
        return this._Salary;
    }
    public set Salary(v : number) {
        this._Salary = v;
    }
    
    
}

File: api/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api.Models
{
  public class Employee
  {
    public int Id { get; set; }

    public string Name { get; set; }

    public decimal Salary { get; set; }

    public int Age { get; set; }

    public int ProjectId { get; set; }
  }
}
